frameworks/Go/go-mongodb/bash_profile.sh

```diff
@@ -0,0 +1,4 @@
+# Set the root of our go installation
+export GOROOT=${IROOT}/go
```

Comment:
On that note, we just merged the change to get rid of the need for this file entirely (issue #1478). After moving the lines, please remove bash_profile.sh entirely. :)